src/injected/drawing-controller.ts

```diff
@@ -19,12 +15,11 @@ import {
     HTMLIFrameResult,
 } from './frameCommunicators/html-element-axe-results-helper';
 import { FrameMessageResponseCallback } from './frameCommunicators/window-message-handler';
-import { InstanceVisibilityChecker } from './instance-visibility-checker';
+import { RegisterDrawer } from './visualization-type-drawer-registrator';
 import { Drawer } from './visualization/drawer';
-import { DrawerProvider } from './visualization/drawer-provider';
 
 export interface VisualizationWindowMessage {
-    visualizationType: VisualizationType;
+    visualizationType?: VisualizationType;
```

Comment:
This is actually not used anywhere within the file; I did not want to remove it in this PR to simplify it. Will remove in a clean-up PR in the future.